lifecycle-management-ui/Ahk.Lifecycle.Management.Ui/Pages/Index.razor

```diff
@@ -1,14 +1,13 @@
 @page "/"
-@inject HttpClient Http
-@inject Microsoft.Extensions.Configuration.IConfiguration config
+@inject IHttpClientFactory httpClientFactory
 
 <PageTitle>Lifecycle Management UI</PageTitle>
 
 <MudPaper Class="pa-4 mb-4">
-    <MudTextField Class="mb-4" @bind-Value="baseUrl" Label="Base url" Variant="Variant.Outlined"></MudTextField>
+    <MudTextField Class="mb-4" @bind-Value="functionName" Label="Function name" Variant="Variant.Outlined"></MudTextField>
```

Comment:
A function nevet nem kell bekérni, az fix, C# kódba beégetett. Csak a baseUrl változhat, az meg konfigurációból jön. Tehát az URL összeállításához _semmit_ nem kérünk a felhasználótól.

Reply:
done